doc/user-guide/hierarchical-data.rst

```diff
@@ -644,3 +646,163 @@ We could use this feature to quickly calculate the electrical power in our signa
+
+.. ipython:: python
+
+    dt.std(dim="time")
+
+.. _coordinate-inheritance:
+
+Coordinate Inheritance
+~~~~~~~~~~~~~~~~~~~~~~
+
+Notice that in the trees we constructed above (LINK OR DISPLAY AGAIN?) there is some redundancy - the ``lat`` and ``lon`` variables appear in each sibling group, but are identical across the groups.
+
+We can use "Coordinate Inheritance" to define them only once in a parent group and remove this redundancy, whilst still being able to access those coordinate variables from the child groups.
+
+.. note::
+    This is also a new feature relative to the prototype `xarray-contrib/datatree <https://github.com/xarray-contrib/datatree>`_ package.
+
+Let's instead place only the time-dependent variables in the child groups, and put the non-time-dependent ``lat`` and ``lon`` variables in the parent (root) group:
+
+.. ipython:: python
+
+    dt = xr.DataTree.from_dict(
+        {
+            "/": ds.drop_dims("time"),
+            "daily": ds_daily.drop_vars(["lat", "lon"]),
+            "weekly": ds_weekly.drop_vars(["lat", "lon"]),
+            "monthly": ds_monthly.drop_vars(["lat", "lon"]),
+        }
+    )
+    dt
+
+This is preferred to the previous representation because it now makes it clear that all of these datasets share common spatial grid coordinates.
+Defining the common coordinates just once also ensures that the spatial coordinates for each group cannot become out of sync with one another during operations.
+
+We can still access the coordinates defined in the parent groups from any of the child groups as if they were actually present on the child groups:
+
+.. ipython:: python
+
+    dt.daily.coords
+    dt["daily/lat"]
+
+(TODO: the repr of ``dt.coords`` should display which coordinates are inherited)
+
+As we can still access them, we say that the ``lat`` and ``lon`` coordinates in the child groups have been "inherited" from their common parent group.
+
+If we print just one of the child nodes, it will still display inherited coordinates, but explicitly mark them as such:
+
+.. ipython:: python
+
+    print(dt["/daily"])
+
+This helps to differentiate which variables are defined on the datatree node that you are currently looking at, and which were defined somewhere above it.
+
+We can also still perform all the same operations on the whole tree:
+
+.. ipython:: python
+    :okexcept:
+
+    dt.sel(lat=[75], lon=[300])
+
+    dt.std(dim="time")
+
+(TODO: The second one fails due to https://github.com/pydata/xarray/issues/8949)
+
+.. _overriding-inherited-coordinates:
+
+Overriding Inherited Coordinates
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We can override inherited coordinates with newly-defined ones, as long as those newly-defined coordinates also align with the parent nodes.
+
+EXAMPLE OF THIS? WOULD IT MAKE MORE SENSE TO USE DIFFERENT DATA TO DEMONSTRATE THIS?
+
+EXAMPLE OF INHERITING FROM A GRANDPARENT?
+
+EXPLAIN DEDUPLICATION?
```

Comment:
Is the plan to include these points in this PR, or merge what is here (maybe with this commented out) and then add more content later?

Reply:
I was going to do it in this PR, but given that everyone seems to be happy with what's here already, and this is a natural break point, perhaps I will just merge this for now.

Reply:
A follow-up issue could be to "document the subtleties of coordinate inheritance"

Reply:
(I had hoped to add these bits before you reviewed it @owenlittlejohns )

Reply:
Removed that content for use in a future PR in https://github.com/pydata/xarray/pull/9501/commits/6cab6f86b61a6b63c8acadda89a6c6cb018d3312